README.md

```diff
@@ -1,26 +1,25 @@
 # Zendesk Apps Tools
 
-These tools make it easy to develop [Zendesk Apps](http://developer.zendesk.com/documentation/apps/).
+## Description
+Zendesk Apps Tools (ZAT) is the number one tool for developing Zendesk Apps. It makes it easy to develop [Zendesk Apps](http://developer.zendesk.com/documentation/apps/). We have a [guide](http://developer.zendesk.com/documentation/apps/reference/tools.html) that explains how to use this tool.
 
-# How to use
+## Owners
+This repo is owned and maintained by the Zendesk Apps team. You can reach us on vegemite@zendesk.com. We are located in Melbourne!
 
-Please refer to [our guide](http://developer.zendesk.com/documentation/apps/reference/tools.html).
+## Getting Started
+`bundle install` is really all that is needed to get it going.
```

Comment:
I think a lot of open source projects tend to have a `Using` heading and a `Hacking` heading (which includes testing and how to contribute), we could do something along those lines.